src/controllers/categoriesController.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from 'express';
+import { Category, CategoryCreationAttributes } from '../database/models/Category';
+import logger from '../logs/config';
+
+export const createCategory = async (req: Request, res: Response) => {
+  try {
+    const { name, description } = req.body as CategoryCreationAttributes;
+    const newCategory = await Category.create({
+      name,
+      description,
+    });
+    res.status(201).json({ ok: true, message: 'New category created successully!', data: newCategory });
+  } catch (error) {
+    if (error instanceof Error) {
```

Comment:
consider using the error handler defined in validations/index file, it catches those errors and send response promptly. we are practicing DRY....

Reply:
Will do